buildenv/jenkins/JenkinsfileBase

```diff
@@ -333,6 +333,8 @@ def createJob( TEST_JOB_NAME, ARCH_OS ) {
 
 	def templatePath = 'openjdk-tests/buildenv/jenkins/testJobTemplate'
 
+	jobParams.put('LIGHT_WEIGHT_CHECKOUT', LIGHT_WEIGHT_CHECKOUT)
```

Comment:
@llxia this change is still here. Do you want the whitespace changes in a separate pr?

Reply:
ah, sorry, I missed this line. Yes, please keep the whitespace changes.